### ISSUE TITLE
Attempting to use uninitialized value Variable

### ISSUE THREAD

**Issue by dongfeicui** [wenyyu/Image-Adaptive-YOLO#76]
[solution1](https://stackoverflow.com/questions/36007883/tensorflow-attempting-to-use-uninitialized-value-in-variable-initialization) and [solution2](https://www.cnblogs.com/lliuye/p/10501744.html)

[batchnorm 1](https://blog.csdn.net/Strive_For_Future/article/details/108375682)
[batchnorm  2](https://blog.csdn.net/Strive_For_Future/article/details/108345162)
